sp800-63c/index.md

```diff
@@ -8,38 +8,6 @@ description: "SP 800-63C"
 
 This is a working draft of NIST Special Publication 800-63C, *Assertions and Federation*. This document is a sub-document referenced by [SP 800-63-3](../sp800-63-3/) covering the associated topics that had been previously in SP 800-63-2.
```

Comment:
I'm not sure we even reference index.md from -63a, -63b, or -63c anywhere. In -3, index.md is the "home page" for the project. So these changes are fine, but we should perhaps get rid of these files entirely (or change them to redirects to the normal views) in the longer run.